spec/components/schemas/content/voice/call.ts

```diff
@@ -0,0 +1,60 @@
+import { SchemaObject } from 'openapi3-ts';
+import { ref as baseRef } from './../base';
+import { ref as ttsRef } from '../tts';
+import { ref as audioFileRef } from '../audiofile';
+import { createComponentRef } from '../../../../../utils/ref';
+
+const text: SchemaObject = {
+  type: 'object',
+  allOf: [{
+    $ref: baseRef,
+  }, {
+    type: 'object',
+    properties: {
+      type: {
+        type: 'string',
+        example: 'call',
+      },
+      audio: {
+        description: 'An array of audio actions',
```

Comment:
Sei que é actions internamente, mas acho que pode ser alguma outra coisa, tipo,
"A sequence of audio bits forming an message to be played on the start of the call"
"A sequence of recorded and generated audio to be played as an message on the start of the call"
"Audio bits played in sequence as a single message on the start of the call"
"Used to compose a audio message played on the start of the call"